helpme/api/views.py

```diff
@@ -14,8 +14,11 @@ class CreateTicketAPIView(LoginRequiredMixin, TicketMetaMixin, CreateAPIView):
 
     def perform_create(self, serializer):
         user_meta = self.get_ticket_request_meta(self.request)
-        
-        instance = serializer.save(user=self.request.user, site=Site.objects.get_current(), user_meta=user_meta)
+
+        current_site = Site.objects.get_current()
```

Comment:
You'd only want to call "Site.objects.get_current()" if self.request.site is null.  Probably better to be an 'else'.  This call hits the DB.

Reply:
Actually may want to move it into a function since it seems to be used in the code 3x.